### PR TITLE
Removed deprecated parameter in class Cart

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Removed deprecated parameter in class Cart
| Type?             | refacto
| Category?         | BO
| BC breaks?        | yes
| Deprecations?     | no
| How to test?      | CI is :green_circle: & Nightly is :green_circle: : https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/4407106258
| Fixed ticket?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

## BC Breaks 
* Remove parameter 2 `$exclusive` for the method `isAllProductsInStock` in class `Cart`

### CODE REVIEW

**Review by FabienPapet — APPROVED** [review]

**Review by 0x346e3730 — APPROVED** [review]

**Review by matthieu-rolland — COMMENTED** [review]
@Progi1984 if someone from the automation team can confirm the test breaking is not related to this PR, we can merge your PR :+1:

**Comment by Progi1984** [comment]
@matthieu-rolland The CI is now :green_circle:

**Comment by lartist** [comment]
Sorry I moved to fast. I shouldn't have merged it before QA approval. Mea culpa

**Pull request merged**
